api/bucket.go

```diff
@@ -108,15 +110,10 @@ func ListBuckets(c *gin.Context) {
 	}
 
 	switch cloudType {
-	case pkgProviders.Alibaba, pkgProviders.Amazon:
-		location, ok := ginutils.RequiredQueryOrAbort(c, "location")
-		if !ok {
-			logger.Debug("missing location")
-
-			return
-		}
-
-		objectStoreCtx.Location = location
+	case pkgProviders.Amazon:
```

Comment:
We should fall back to the default location only when no "location" query param is provided by the caller.